### ISSUE TITLE
coredump during gossip - few seconds after nodetool toppartitions

### ISSUE THREAD

**Issue by fgelcer** [scylladb/scylla#4765]
it is still unclear exactly what happened, as there is not backtrace.
the job running was the longevity 4h 100Gb, running against scylla 3.1

```
systemd-coredump: Process 4000 (scylla) of user 996 dumped core.#012#012Stack trace of thread 4013:#012#0  0x00000000023ea157 _ZNKSt8__detail18_Mod_range_hashingclEmm (/opt/scylladb/libexec/scylla.bin)
```

```
           PID: 4000 (scylla)
           UID: 996 (scylla)
           GID: 1001 (scylla)
        Signal: 8 (FPE)
     Timestamp: Sun 2019-07-28 04:31:48 UTC (1min 54s ago)
  Command Line: /usr/bin/scylla /opt/scylladb/bin/../libexec/scylla.bin --blocked-reactor-notify-ms 500 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc --log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix --io-properties-file=/etc/scylla.d/io_properties.yaml --cpuset 1-7,9-15
    Executable: /opt/scylladb/libreloc/ld.so
 Control Group: /
       Boot ID: e1092b1e865b49ef8664214f4fc34995
    Machine ID: df877a200226bc47d06f26dae0736ec9
      Hostname: ip-172-30-0-204.ec2.internal
      Coredump: /var/lib/systemd/coredump/core.scylla.996.e1092b1e865b49ef8664214f4fc34995.4000.1564288308000000
       Message: Process 4000 (scylla) of user 996 dumped core.
                
                Stack trace of thread 4013:
                #0  0x00000000023ea157 _ZNKSt8__detail18_Mod_range_hashingclEmm (/opt/scylladb/libexec/scylla.bin)
```

logs can be found [here](https://cloudius-jenkins-test.s3.amazonaws.com/07a39896-5591-4e12-98fe-132c02365588/job_log.zip)

**Comment by avikivity** [comment]
Always specify the version and commit hash in bug reports.

**Comment by avikivity** [comment]
Closing, not enough info. May be fixed by b0d231262385d8a56fbfe2c1ea34b9496bab223f.
